src/theory/quantifiers/trigger.h

```diff
@@ -156,13 +242,14 @@ class TriggerTrie {
 public:
```

Comment:
This class needs better documentation.  The class comment should explain what exactly the trie data are.  Each method should also be documented.